app/overrides/profiles_noindex.rb

```diff
@@ -0,0 +1,17 @@
+# frozen_string_literal: true
+
+##
+# Snippet for insert the meta tag 'noindex' in the head section of the views listed below
+# using Deface gem
+##
+views = ["decidim/profiles/show", 
+         "decidim/user_activities/index",
+         "decidim/user_timeline/index",
+         "decidim/user_conversation/index"].freeze
+
+views.each do |view|
+  Deface::Override.new(:virtual_path => view,
+                       :name => "profiles_noindex",
+                       :insert_after => "erb[loud]:contains('cell')",
```

Comment:
Not sure it will insert only once, case you have two `cell` used.

Reply:
OK, I've improved the criteria for `:insert_after => "erb[loud]:contains('cell \"decidim/profile\"')",` ;)